packages/core/package.json

```diff
@@ -16,6 +16,9 @@
     "type": "module",
     "main": "./dist/index.js",
     "types": "./dist/index.d.ts",
+    "files": [
+        "/dist"
```

Comment:
In other project, i've seen this a lot: 

```
    "files": [
        "dist",
        "CHANGELOG.md",
        "README.md"
    ],
```

you ship the code, but also the documentation that can be read if needed